user/db.js

```diff
@@ -6,7 +6,11 @@ db.exec(`CREATE TABLE IF NOT EXISTS users (
         id INTEGER PRIMARY KEY AUTOINCREMENT,
         name TEXT NOT NULL UNIQUE COLLATE NOCASE,
         password TEXT NOT NULL,
-        versionkey INTEGER NOT NULL DEFAULT 1
+        versionkey INTEGER NOT NULL DEFAULT 1,
+        email TEXT NOT NULL UNIQUE COLLATE NOCASE,
+        created_on DATETIME DEFAULT CURRENT_TIMESTAMP,
+        recovery_email TEXT NOT NULL COLLATE NOCASE,
```

Comment:
interesting choice to make phone and recovery not null. Not sure that would be good for users. Maybe not even email (not docking points, just explaiing)